ui_framework/dist/ui_framework.css

```diff
@@ -706,6 +706,7 @@ body {
   /* 2 */
   font-size: 10px !important;
   /* 2 */
+  -webkit-transition: background-color 0.1s linear;
```

Comment:
There's something funny going on where some people have been generating CSS that adds these properties, and yet I've been generating CSS that removes them. Are you using yarn to install your NPM deps by any chance? Which version of postcss do you have installed? I have 5.2.17.